src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -424,7 +429,41 @@ BtreeLBAManager::BtreeLBAManager(
   SegmentManager &segment_manager,
   Cache &cache)
   : segment_manager(segment_manager),
-    cache(cache) {}
+    cache(cache)
+{
+  register_metrics();
+}
+
+LBABtree::lba_tree_inner_stats_t LBABtree::lba_tree_inner_stats;
+void BtreeLBAManager::register_metrics()
+{
+  namespace sm = seastar::metrics;
+  auto tree_label = sm::label("tree");
+  auto lba_label = tree_label("LBA");
+  metrics.add_group(
+    "LBA",
+    {
+      sm::make_counter(
+        "lba_tree_alloc_extents",
```

Comment:
The final metric name will look like "LBA_lba_tree_alloc_extents" with label "tree=LBA".

So I think we can remove "lba_tree_" from the metric names, also remove the "tree=LBA" label as they are all duplicated information.